custom-targets/infrastructure-manager/quickstart/QUICKSTART.md

```diff
@@ -0,0 +1,191 @@
+
+1. `clouddeploy.jobRunner` role:
+
+    ```shell
+    gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/clouddeploy.jobRunner"
+    ```
+
+2. `iam.serviceAccountUser` role, to give the service account `actAs` permission:
+
+    ```shell
+    gcloud iam service-accounts add-iam-policy-binding $(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/iam.serviceAccountUser" \
+    --project=$PROJECT_ID
+    ```
+
+3. `config.agent` role:
+
+    ```shell
+    gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/config.agent"
+    ```
+
+4. `config.admin` role:
+
+    ```shell
+    gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/config.admin"
+    ```
+
+5. `compute.networkAdmin` role:
+
+    ```shell
+    gcloud projects add-iam-policy-binding $PROJECT_ID \
+    --member=serviceAccount:$(gcloud projects describe $PROJECT_ID \
+    --format="value(projectNumber)")-compute@developer.gserviceaccount.com \
+    --role="roles/compute.networkAdmin"
+    ```
+
+## 4. Build and Register a Custom Target Type for Infrastructure Manager
+From within the `quickstart` directory, run the following command to build the Infrastructure Manager deployer image and register a Cloud Deploy custom target type that references the image:
+
+```shell
+../build_and_register.sh -p $PROJECT_ID -r $REGION
+```
+
+For information about the `build_and_register.sh` script, see the [README](../README.md#build)
+
+## 5. Create delivery pipeline and targets
+Replace the placeholders in the `clouddeploy.yaml` in this directory with your environment variable values:
+
+```shell
+sed -i "s/\$PROJECT_ID/${PROJECT_ID}/g" ./clouddeploy.yaml
+sed -i "s/\$REGION/${REGION}/g" ./clouddeploy.yaml
+```
+
+Apply the Cloud Deploy configuration:
+
+```shell
+gcloud deploy apply --file=clouddeploy.yaml --project=$PROJECT_ID --region=$REGION
+```
+
+## 6. Create a release
+Create a Cloud Deploy release for the configuration defined in the `configuration` directory. This will automatically create a rollout that deploys the Terraform configuration to the dev target.
+
+```shell
+gcloud deploy releases create release-001 --delivery-pipeline=im-network-pipeline --source=configuration --project=$PROJECT_ID --region=$REGION
```

Comment:
nit, use time/date syntax so this command can be re-run

Reply:
Going to keep as-is for now but can revisit this. We don't currently do this in the other samples nor our official docs since it adds a bit more difficulty to constructing the rest of the commands.